app/views/authenticatedFullUserReactTemplate.scala.html

```diff
@@ -0,0 +1,35 @@
+@import assets.AssetsResolver
+@import com.gu.identity.play.IdUser
+@(title: String, id: String, js: String, user: IdUser)(implicit assets: AssetsResolver)
+
+@scripts = {
+    <script type="text/javascript">
+        window.guardian = window.guardian || {};
+        window.guardian.user = {
+            email: "@user.primaryEmailAddress",
+            @user.publicFields.displayName.map { displayName =>
+            displayName: "@displayName",
+            }
+            @for(fields <- user.privateFields; firstName <- fields.firstName; lastName <- fields.secondName) {
+            firstName: "@firstName",
+            lastName: "@lastName",
+            }
+        };
+    </script>
+    <script type="text/javascript" src="@assets(js)"></script>
+}
+
+@main(title = title, scripts = scripts) {
+    <main id="@id">
+        <noscript>
+            <div style="text-align: center;
```

Comment:
If we are going to start to repeat the no js message in different pages, propably is better to put it inside a twirl fragment.